modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,668 @@
+--
+.Input
+[source,bash]
+----
+/usr/bin/redpanda --version
+----
+
+.Output
+[,bash,role=no-copy,subs="attributes+"]
+----
+{full-version} - {latest-release-commit}
+----
+--
+
+[[cpu-memory]]
+=== Correct CPUs and Memory configured
+
+Ensure that you have the  xref:deploy:deployment-option/self-hosted/manual/production/requirements.adoc#cpu-and-memory[correct number of CPUs and sufficient memory] to run Redpanda.
+
+[.side-by-side]
+--
+.Input
+[source,bash]
+----
+journalctl -u redpanda | grep "System resources"
+----
+
+.Output
+[source,bash]
+----
+Mar 25 12:16:18 ip-172-31-10-199 rpk[3957]: INFO  2024-03-25 12:16:18,105 [shard 0:main] main - application.cc:350 - System resources: { cpus: 8, available memory: 55.578GiB, reserved memory: 3.890GiB}
+----
+--
+
+[[mounted-disks]]
+=== Confirm disks are correctly mounted
+
+Ensure that the correct disks are mounted and if multiple devices are used, they are configured as RAID-0. Other RAID configurations can have significantly worse latencies. The file system should be of type XFS. If XFS is unavailable, ext4 is an appropriate alternative. 
+
+[.side-by-side]
+--
+.Input
+[source,bash]
+----
+grep data_directory /etc/redpanda/redpanda.yaml
+    data_directory: /var/lib/redpanda/data
+df -khT /var/lib/redpanda/data
+----
+
+.Output for NVMe with XFS
+[source,bash]
+----
+Filesystem     Type  Size  Used Avail Use% Mounted on
+/dev/nvme0n1   xfs   1.8T   14G  1.8T   1% /mnt/vectorized
+----
+
+.Output for madm RAID mount point
+[source,bash]
+----
+Filesystem     Type  Size  Used Avail Use% Mounted on
+/dev/md0       xfs    14T   99G   14T   1% /mnt/vectorized
+----
+--
+
+
+The following example shows how to obtain further details about the RAID array.
+
+[.side-by-side]
+--
+.Input
+[source,bash]
+----
+mdadm --detail /dev/md0
+----
+
+.Output 
+[source,bash]
+----
+/dev/md0:
+           Version : 1.2
+     Creation Time : Thu Apr 18 11:03:46 2024
+        Raid Level : raid0
+        Array Size : 14648172544 (13969.59 GiB 14999.73 GB)
+      Raid Devices : 2
+     Total Devices : 2
+       Persistence : Superblock is persistent
+
+       Update Time : Thu Apr 18 11:03:46 2024
+             State : clean
+    Active Devices : 2
+   Working Devices : 2
+    Failed Devices : 0
+     Spare Devices : 0
+
+            Layout : -unknown-
+        Chunk Size : 512K
+
+Consistency Policy : none
+
+              Name : ip-172-31-24-82:0  (local to host ip-172-31-24-82)
+              UUID : e9574118:10d562bf:ed3ca2d9:68ccc3a6
+            Events : 0
+
+    Number   Major   Minor   RaidDevice State
+       0     259        2        0      active sync   /dev/nvme2n1
+       1     259        0        1      active sync   /dev/nvme1n1
+----
+--
+
+Use the results here to verify that the expected disks are present and that the RAID level is set as expected (typically, this would be `raid0` in a production system as data resilience is provided by Raft across Redpanda brokers, rather than by RAID).
```

Comment:
```suggestion
Use these results to verify that the expected disks are present and the expected RAID level is set. (Typically, this would be `raid0` in a production system, as data resilience is provided by Raft across Redpanda brokers, rather than by RAID.)
```